setup.py

```diff
@@ -43,22 +44,12 @@ def has_environment_marker_support():
     try:
         return pkg_resources.parse_version(setuptools.__version__) >= pkg_resources.parse_version('0.7.2')
     except Exception as exc:
-        sys.stderr.write("Could not test setuptool's version: %s\n" % exc)
-        return False
+        raise SystemExit("setuptools does not yet support markers: %s\n" % exc)
```

Comment:
I think we should give a more helpful message here: `"setuptools version is %s, which does not support markers. Please update your setuptools version"` or so.